### ISSUE TITLE
Unable to reproduce egs2/swbd results

### ISSUE THREAD

**Issue by CuiMingyu** [espnet/espnet#4602]
Dear sir,

I use the config https://github.com/espnet/espnet/blob/a65cc78de7e18c867f4be5fc0b9b695875c78c70/egs2/swbd/asr1/conf/tuning/train_asr_conformer2.yaml

but the ctc loss and att loss can not converge. So I changed the warmup lr from 0.006 to 0.002 and I get the results below:
swb: 8.4
callhm: 15.4

Above result didn' t use sp but get the similar result with https://github.com/espnet/espnet/blob/a65cc78de7e18c867f4be5fc0b9b695875c78c70/egs2/swbd/asr1/RESULTS.md#asr_train_asr_cformer5_raw_bpe2000_sp.
How can I make sure that the result from mine can be believed?

Also, https://github.com/espnet/espnet/blob/a65cc78de7e18c867f4be5fc0b9b695875c78c70/egs2/swbd/asr1/RESULTS.md#fbank-pitch-model-with-bpe2k-specaug--with-and-without-12l-bpe2k-swbdfisher-transformer-lm

the model page and config yaml can not found. Could you upload them again? 

Could you publish the config yaml that can reproduce the results of the page?

Thank you very much!

**Comment by sw005320** [comment]
It is very difficult to reproduce the same results since the result would be changed with your environments (GPU type, number of GPUs, cuda/cudnn versions, pytorch versions, etc.).

sp would not improve the performance so much, and I think your number looks good and you can believe your results.

@roshansh-cmu, can you put the config files?
You may also upload your model to the HF hub.

**Comment by CuiMingyu** [comment]
Hi, it is difficult for me to reproduce similar results based on above setup.

Could you give me the setup below? I have turned several lr and can' t get the 10.9 result.
https://github.com/espnet/espnet/blob/a65cc78de7e18c867f4be5fc0b9b695875c78c70/egs2/swbd/asr1/RESULTS.md#fbank-pitch-model-with-bpe2k-specaug--with-and-without-12l-bpe2k-swbdfisher-transformer-lm

Thank you very much!

**Comment by roshansh-cmu** [comment]
Our pretrained model has been uploaded to the HF Repo [here](https://huggingface.co/espnet/roshansh_asr_base_sp_conformer_swbd)

**Comment by roshansh-cmu** [comment]
The LM Config file is [here](https://github.com/espnet/espnet/blob/master/egs2/swbd/asr1/conf/tuning/train_lm_transformer2.yaml).

**Comment by CuiMingyu** [comment]
OK, thanks. I will try it.
